jdi-light/src/main/java/com/epam/jdi/light/settings/WebSettings.java

```diff
@@ -242,7 +242,32 @@ private static PageLoadStrategy getPageLoadStrategy(String strategy) {
     }
     public static Properties getProperties(String path) {
         // TODO use mergePath macos and windows
-        Properties p = PropertyReader.getProperties("/../../target/classes/" + path);
-        return p.size() > 0 ? p : PropertyReader.getProperties(path);
+        File file = new File(path);
+        System.out.println("ffffffff: " + file.getAbsolutePath());
+        System.out.println("ffffffff: " + new File("./").getAbsolutePath());
+        if (file.getAbsolutePath().contains("chrome")) {
+            if (file.exists()) {
+                System.out.println("EXIT 0");
+                //   System.exit(0);
+            } else {
+                System.out.println("EXIT 1");
+                System.exit(1);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [System.exit() should not be used in J2EE/JEE apps](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4253331)